rts/Lua/LuaMathExtra.cpp

```diff
@@ -184,6 +185,34 @@ int LuaMathExtra::smoothstep(lua_State* L) {
 	return 1;
 }
 
+
+/*** Returns the normalize vector of an given vector.
+ *
+ * @function math.normalize
+ * @number x
+ * @number y
+ * @number z
+ * @treturn nil|normalize vector
+ */
+int LuaMathExtra::normalizevector(lua_State* L)
+{
+	RECOIL_DETAILED_TRACY_ZONE;
+	lua_Number res = 0.0f;
+
+	for (int i = lua_gettop(L); i >= 1; i--) {
+		res += Square(luaL_checknumber_noassert(L, i));
+	}
+	res = math::sqrt(res);
+	if (res == 0)
+		return 0;
+	for (int i = lua_gettop(L); i >= 1; i--) {
+		float tmp = luaL_checknumber_noassert(L, i);
+		tmp = res / tmp;
```

Comment:
Looks incorrect.

Reply:
You're right it should be tmp = tmp / res. It fix now